docs/customization/customizing-your-storefront.mdx

```diff
@@ -0,0 +1,170 @@
+---
+sidebar_position: 1
+---
+
+# Customizing your storefront
+
+In this guide, we're going to show you an example of how to customize your storefront. In
+particular, we're going to add a hero image to the hero section of the home page:
+
+![Hero image placeholder](/img/customizing-your-storefront/hero-image-placeholder.png)
+
+## Installing the frontend
+
+With Solidus 3.2, installing Solidus would automatically install[SolidusStarterFrontend][], which is
+the new frontend solution for Solidus. As a Rails application template, it copies all the Solidus
+frontend code directly to your Rails app. This makes the frontend code directly accessible to your
+application. If you're familiar with customizing views and assets on a regular Rails app, you
+already have the knowledge to customize the frontend of a Solidus application.
+
+For the rest of the guide, we'll assume that you have a running Solidus application with
+SolidusStarterFrontend as your storefront.
+
+## Editing a view
+
+If we inspect the home page, we'll see that the hero container has the following HTML code:
+
+```html
+<div class="hero__container">
+  <div class="hero__message">
+    <h1 class="hero-title">
+      The only eCommerce platform you’ll ever need.
+    </h1>
+
+    <p class="subtitle">
+      Build, customize and scale your store with no limits or license fees.
+      Solidus is the free, open-source eCommerce framework for
+      digitally-native brands, fast-growing online businesses and pragmatic
+      developers.
+    </p>
+  </div>
+</div>
+```
+
+If we search for views with the CSS class `"hero__container"` in your Solidus app, we find
+`app/views/spree/home/_hero.html.erb`. This partial was copied to your app during the installation
+of SolidusStarterFrontend.
+
+Let's use the [`image_tag`][image_tag] helper to try and add a placeholder for the hero image in
+that partial:
+
+```html
+<div class="hero__container">
+  <%= image_tag('hero-image') %>
```

Comment:
What we've been doing in other places when modifying code is adding the diff for it ([example](https://github.com/solidusio/edgeguides/blob/338608abf9f0101c0717f5acb121acb00e717f03/docs/customization/customizing-the-backend.mdx?plain=1#L41)). WDYT?

Reply:
Sounds good. I'll update the code using diff :+1: